automation-api/lib/pilot/datatypes.py

```diff
@@ -0,0 +1,60 @@
+import json
+from dataclasses import dataclass
+from typing import List
+
+
+@dataclass
+class QuestionOption:
+    letter: str
+    option_text: str
+    correctness: int
+
+    def letter_and_text(self) -> str:
+        return f"{self.letter}. {self.option_text}"
+
+
+@dataclass
+class Question:
```

Comment:
I agree that we should use the pydantic schemas. 
I think the primary issue is that, I have to join 2 tables to get the dataframe containing  both question and options, see https://github.com/Gapminder/gapminder-ai/blob/c8987a812ae9ee75a29af8e099e207b2f88d274c/automation-api/lib/pilot/helpers.py#L72
also model and model config: https://github.com/Gapminder/gapminder-ai/blob/c8987a812ae9ee75a29af8e099e207b2f88d274c/automation-api/lib/pilot/helpers.py#L189

we can store both questions and question options / model and model configs in one table.  But it will result in  some duplication in the table

Reply:
But I think it's still doable if I don't join two tables. Let me make some changes and get back to you